package.json

```diff
@@ -246,7 +246,7 @@
     "react-resize-detector": "^4.2.0",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
-    "react-sizeme": "^2.3.6",
+    "react-sizeme": "2.3.6",
```

Comment:
Why is this necessary?

Ideally we and all the packages in our dependencies would use `^` dependency versions so that yarn can install the same version for as many packages as possible. Putting a type restriction in place like this is indicating that we're not compatible with other versions of `react-sizeme` (and other packages where you've removed the `^` from) which I think is the wrong signal to send. If you want to force packages to install a specific version for now I recommend modifying the yarn.lock file manually (I regularly will delete declaration for specific packages to trigger yarn reresolving them and the other places they're used). If you want to enforce it over time you should use a "resolutions" field in package.json, but that's a pretty heavy handed approach which should be used sparingly.

Unless we're really only compatible with a specific patch version I don't think we should be using this strategy.